hack/dev-up.sh

```diff
@@ -20,29 +20,44 @@ set -o pipefail
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${REPO_ROOT}"/hack/util.sh
 
-KUBECONFIG_PATH=${KUBECONFIG_PATH:-"${HOME}/.kube/kubeadmiral/kubeconfig.yaml"}
+KUBECONFIG_DIR=${KUBECONFIG_DIR:-"${HOME}/.kube/kubeadmiral"}
 HOST_CLUSTER_NAME=${HOST_CLUSTER_NAME:-"kubeadmiral-host"}
 MEMBER_CLUSTER_NAME=${MEMBER_CLUSTER_NAME:-"kubeadmiral-member"}
 MANIFEST_DIR=${MANIFEST_DIR:-"${REPO_ROOT}/config/crds"}
 CONFIG_DIR=${CONFIG_DIR:-"${REPO_ROOT}/config/sample/host"}
 NUM_MEMBER_CLUSTERS=${NUM_MEMBER_CLUSTERS:-"3"}
-
-mkdir -p "$(dirname "${KUBECONFIG_PATH}")"
-
-kind delete clusters "${HOST_CLUSTER_NAME}" $(seq 1 "${NUM_MEMBER_CLUSTERS}" | xargs -I{} echo "${MEMBER_CLUSTER_NAME}-{}")
+export CLUSTER_PROVIDER=${CLUSTER_PROVIDER:-"kind"}
```

Comment:
Why is `CLUSTER_PROVIDER` exported while the rest are not?

Reply:
If the rest are not required to be exported, can we mark them `local` for more clarity?

Reply:
oops, this does not need to be exported